### ISSUE TITLE
[Alpha 1.16.0.10] Notes Don't Honor Color

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#5558]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

The new Obsidian Notes on Person Profile timeline do not show in the color that is defined on them.

### Expected Behavior

Notes would show with the colors they have defined on the Note Type. Here's an example of my profile timeline in v14.3:

![image](https://github.com/SparkDevNetwork/Rock/assets/3903408/27245bb7-aa25-44a6-a811-b82a552b595d)

### Actual Behavior

All notes are white, regardless of the color defined on the Note Type. Here's the same profile in v16 alpha:

![image](https://github.com/SparkDevNetwork/Rock/assets/3903408/4ca58ade-0105-4f20-9812-a735cdf6bc71)

### Steps to Reproduce

1. Go to a Profile that has notes that should have a certain color as defined in the Note Type
2. Observe that all notes are white.

### Rock Version

1.16.0.10

### Client Culture Setting

en-US

**Comment by jonedmiston** [comment]
This is somewhat a bug and somewhat by design. The design part is that we're hoping the make the notes look more polished and modern. The bug part is that the round dot should probably be the same color you chose.

![image](https://github.com/SparkDevNetwork/Rock/assets/548178/cb0fb341-b6b2-4fbc-9ca4-29313ffc971e)

In our offline discussion though we agree that more emphasis is needed. And have made the dot a label.

![image](https://github.com/SparkDevNetwork/Rock/assets/548178/ea1b7504-41ee-4868-aea9-b9b3ea9b25dc)

Edit made here:
https://github.com/SparkDevNetwork/Rock/commit/4f390ad8a998835a0433d762b0ff3acb956e97c5

Will be fixed in beta.

**Comment by mikedotmundy** [comment]
I do like the new design of this - I like the label vs the dot. However, the colors we had previously (v15.1) did not make it over into v16.0.

For example, this is our instance (v15.1) for the Event Registration Note Type. We have defined only the "Border Color", which causes the dot to be green in this example.
<img width="797" alt="Screenshot 2023-09-12 at 22 00 56" src="https://github.com/SparkDevNetwork/Rock/assets/23226974/4c7904fe-51f3-49ea-9e8d-4ad1f610bc04">

However, when updating to v1.16.0.11, the color field is blank completely. It is like this for all of our Note Types, which means, we will have to go in and edit every Note Type to get the colors we want. And, if I had only updated this on production and not had a dev site, I would have lost what all my colors were upon updating. 
<img width="815" alt="Screenshot 2023-09-12 at 22 02 35" src="https://github.com/SparkDevNetwork/Rock/assets/23226974/0039233e-b723-4f5f-a3a3-4f06f98474a7">

So my question is, should the colors from the "Border Color" (v15.1), be migrated over to the new "Color" (v16.0) field somehow? Or is the v15.1 "Background Color" field meant to migrate the the new v16.0 "Color" field? I'm trying to see if there is a way to preserve what my settings are and not have to manually re-do them all once updating our production site to v16.0.
